### DISCUSSION TITLE
[Feature] 能否设置一个快速全library搜索的快捷键

### DISCUSSION THREAD

**Discussion by fredericky123** [windingwind/zotero-actions-tags#280] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

- OS: Mac
- Zotero Version: 最新
- Plugin Version: 最新

### Describe the feature request

**Is your feature request related to a problem? Please describe.**

**Why do you need this feature?**
A clear and concise description of why you need this feature.

现在的快速搜索 cmd shift k 只能搜索当前collection能否加个快捷，实现全库的检索

### Describe the solution you'd like

**The solution you'd like**

**Alternatives you've considered**

**Comment by Geo123abc** [comment]
Quick search cmd shift k这个在 windows下可以吗？

**Reply by YaoLiMuMu** [reply]
有安装style的话直接Shift+P就可以全局搜索标题了, 这个template里面自带, 应该满足使用
![image](https://github.com/windingwind/zotero-actions-tags/assets/32826811/b7cd98c3-d642-4f87-a45f-4be1b995751f)

**Reply by fredericky123** [reply]
感谢提示 果然好用！

**Comment by pencilheart** [comment]
cmd+shift+F

**Comment by Ghiclgi** [comment]
Here is a solution based on  [[Share\] Return to "My Library"](https://github.com/windingwind/zotero-actions-tags/discussions/430). Just need to move the focus to the search box:
```javascript
/**
 * When assinged to a keybinding, this allows you to switch back to the library view.
 * @author promicrobial
 * @usage 
 * @link https://gist.github.com/promicrobial/c2ce52599aca3c7ca44d9aff2628148a.js
 * @see https://github.com/windingwind/zotero-actions-tags/discussions/
 */

const Zotero = require("Zotero");
const Zotero_Tabs = require("Zotero_Tabs");

if (Zotero_Tabs.selectedID != 'zotero-pane') {
  Zotero_Tabs.select('zotero-pane')
}
Zotero.getActiveZoteroPane().collectionsView.selectLibrary(Zotero.Libraries.userLibraryID);
Zotero.getActiveZoteroPane().document.getElementById('zotero-tb-search').searchTextbox.select();
```

**Comment by rwmath27** [comment]
generated by Claude Opus 4.6
```javascript
const Zotero = require("Zotero");
const Zotero_Tabs = require("Zotero_Tabs");
if (Zotero_Tabs.selectedID != 'zotero-pane') {
  Zotero_Tabs.select('zotero-pane')
}
Zotero.getActiveZoteroPane().collectionsView.selectLibrary(Zotero.Libraries.userLibraryID);
Zotero.getActiveZoteroPane().document.getElementById('zotero-tb-search').searchTextbox.select();
```
